binarySearch/.idea/workspace.xml

```diff
@@ -18,7 +18,7 @@
       <file pinned="false" current-in-tab="true">
         <entry file="file://$PROJECT_DIR$/src/binarySearch.java">
           <provider selected="true" editor-type-id="text-editor">
```

Comment:
Please get rid of all the files except src/* pom.xml. Use .gitignore for the same.

Reply:
Did the changes in the tweeting project. Was able to hide the properties file having my credentials as well.